include/BulkMove.php

```diff
@@ -255,6 +292,28 @@ public function get_translations_path() {
 		return $this->translations_path;
 	}
 
+	/**
```

Comment:
Let's move Loadie implementation to a new branch.

Reply:
> Let's move Loadie implementation to a new branch.

Fixed.

Reply:
@mariadanieldeepak 

I am still seeing `add_loadie` method in the diff. Has it been removed?

Reply:
> I am still seeing add_loadie method in the diff. Has it been removed?

Ah, I missed to remove this method. The Interface was removed already. Fixed now.